dlpy/network.py

```diff
@@ -23,9 +23,11 @@
 from dlpy.layers import Layer
 from dlpy.utils import DLPyError, input_table_check, random_name, check_caslib, caslibify, get_server_path_sep, underscore_to_camelcase
 from .layers import InputLayer, Conv2d, Pooling, BN, Res, Concat, Dense, OutputLayer, Keypoints, Detection, Scale, Reshape
+import dlpy.model
 import collections
 import pandas as pd
 import swat as sw
+from copy import deepcopy
```

Comment:
is this available in version earlier than python 3.6?

Reply:
deepcopy is supported since 2.xx